lib/dns.js

```diff
@@ -25,7 +25,7 @@ function errnoException(err, syscall, hostname) {
   }
   var ex = null;
   if (typeof err === 'string') {  // c-ares error code.
-    ex = new Error(syscall + ' ' + err + (hostname ? ' ' + hostname : ''));
+    ex = new Error(`${syscall} ${err}${(hostname ? ' ' + hostname : '')}`);
```

Comment:
FYI, this will still trigger a "unexpected string concatenation" in ESLint because of the `' ' + hostname` part.